io.gitlab.windstille.Windstille-0_2.json

```
@@ -14,36 +14,24 @@
         "--socket=pulseaudio"
     ],
```

```suggestion
    ],
    "cleanup": [
        "/lib/libguile-3.0.so.1.2.0-gdb.scm"
    ],

```

Adding this avoid the error `ldconfig: /app/lib/libguile-3.0.so.1.2.0-gdb.scm is not an ELF file - it has the wrong magic bytes at the start.`